hw/hw_10_task_01.py

```diff
@@ -0,0 +1,152 @@
+import asyncio
+import json
+import re
+from typing import Dict, List
+
+import aiohttp
+
+from bs4 import BeautifulSoup
+
+import requests
```

Comment:
`aiohttp` and `requests` doing the same thing. In general, it is not recommended to use them simultaneously.